app/views/payments/WhatYouOwe.scala.html

```diff
@@ -166,12 +166,14 @@ <h3 class="govuk-heading-s">@messages("whatYouOwe.agent.isIncorrectOne")</h3>
         </p>
         <p class="govuk-body">@messages("whatYouOwe.agent.isIncorrectFour")</p>
       } else {
-        <h2 id="cannot-pay-heading" class="govuk-heading-s">@messages("whatYouOwe.ifYouCannotPayOne")</h2>
-        <p class="govuk-body">@messages("whatYouOwe.ifYouCannotPayTwo")
-          <a class="govuk-link" target="_blank" rel="noreferrer noopener" href="@appConfig.govUKDifficultiesPayingUrl">
-            @messages("whatYouOwe.ifYouCannotPayThree")</a>.
-          @messages("whatYouOwe.ifYouCannotPayFour")
-        </p>
+        @if(appConfig.features.overdueTimeToPayDescriptionEnabled() && model.containsOverduePayments) {
```

Comment:
You're missing this logic for the corresponding agent section too. This is the progressive disclosure "Payment help" - same thing, we only want to show it when this logic is true.